app/assets/stylesheets/components/_index.scss

```diff
@@ -1 +1,3 @@
 @import "_navbar";
+@import "_cards";
+@import "_buttons";
```

Comment:
This creates a conflict because the file has been updated on master.
Try doing that instead
```
@import "buttons"
@import "cards"
@import "form";
@import "navbar";
```

notice that the underscore is not needed when doing the import